src/taskgraph/generator.py

```diff
@@ -31,22 +31,47 @@ class KindNotFound(Exception):
     """
 
 
+# In an ideal world these would be part of each loader. Unfortunately, loaders
+# can only provide information as part of yielding task inputs, and we need to
+# know the set of transforms prior to starting to read from the loader.
+DEFAULT_TRANSFORMS = [
+    "taskgraph.transforms.job:transforms",
+    "taskgraph.transforms.task:transforms",
```

Comment:
Maybe this could go in the graph config instead?

Reply:
I do like the idea of a project being able to override or append to this - but especially if we're going with the `default` transform idea it feels like this belongs in taskgraph to make it more "batteries included".